arsceneview/src/main/java/io/github/sceneview/ar/node/ArModelNode.kt

```diff
@@ -41,7 +41,6 @@ open class ArModelNode : ArNode {
             doOnAttachedToScene { sceneView ->
                 (sceneView as? ArSceneView)?.apply {
                     planeFindingMode = value.planeFindingMode
-                    depthEnabled = value.depthEnabled
                     instantPlacementEnabled = value.instantPlacementEnabled
                 }
             }
```

Comment:
Remove the whole setter. And add a notice in the field doc to tell that the feature must be enabled at SceneView level